todo-frontend-angular/src/app/todo/todolist/todolist.component.ts

```diff
@@ -1,15 +1,33 @@
 import { Component, OnInit } from '@angular/core';
+import { Todo } from 'src/app/models/todo.model';
+import { TodoService } from '../todo.service';
 
 @Component({
   selector: 'app-todolist',
   templateUrl: './todolist.component.html',
   styleUrls: ['./todolist.component.scss']
 })
 export class TodolistComponent implements OnInit {
-
-  constructor() { }
+  todos: Todo[] = [];
+  textBtnConfig = {
```

Comment:
Should have some sort of type. Is a type can't be found. Then it should be of type Unknown